test/Factories.hs

```diff
@@ -37,8 +37,8 @@ createUser ident = do
     Entity planId _ <- createFreePlan
 
     insertEntity User
-        { userName = "John Smith (" ++ ident ++ ")"
-        , userEmail = "john-" ++ ident ++ "@gmail.com"
+        { userName = "Dummy Login"
+        , userEmail = "dummy@example.com"
```

Comment:
> It seems like using the dummy plugin will result in the dummy profile, regardless of the provided login details.

There are no "provided login details". When you login via Dummy you only specify ident, not name/email or anything like that. The issue is that any values already on an existing user are overwritten.

> Does it actually matter what these values are?

It matters that it matches what's on the `dummyProfile` side. This is because in the `Handler.Comment` spec, we assert on a response that includes user name/email. Therefore, the factory user here (used as expected) has to have values matching the `dummyProfile` (which ultimately ends up on actual)

Does that make sense?